docs/sbpy/data.rst

```diff
@@ -349,10 +349,12 @@ Mauna Kea Observatory (IAU observatory code ``568``) from the `JPL Horizons serv
     >>> print(eph) # doctest: +ELLIPSIS
     <sbpy.data.ephem.Ephem object at ...>
     >>> print(eph.table)
+
```

Comment:
... those doctests can be really annoying. Is it possible that your IDE added blank lines after the example code blocks? Pytest will interpret these as 'no output'. Could you please remove them manually?